client/cypress/integration/sessions.spec.js

```diff
@@ -61,28 +61,9 @@ describe("sessions screen", () => {
     tableHead.get("th").eq(6).should("contain", "Username");
   });
 
-  it("displays a list of ungraded sessions by default", () => {
+  it("displays a list of sessions", () => {
     cy.visit("/sessions");
     const tableBody = cy.get("table tbody");
-    tableBody.get("tr").should("have.length", 2);
-    cy.get("table>tbody>tr:nth-child(1)>td:nth-child(1)").should(
-      "contain",
-      "lesson 2"
-    );
-    cy.get("table>tbody>tr:nth-child(1)>td:nth-child(4)").should(
-      "contain",
-      "50"
-    );
-    cy.get("table>tbody>tr:nth-child(1)>td:nth-child(3)").should(
-      "contain",
-      "?"
-    );
-  });
-
-  it("toggles a list of graded and ungraded session", () => {
-    cy.visit("/sessions");
-    cy.get("#toggle").click();
-    const tableBody = cy.get("table tbody");
     tableBody.get("tr").should("have.length", 3);
     cy.get("table>tbody>tr:nth-child(1)>td:nth-child(1)").should(
```

Comment:
I know not new, but how hard would be to give each row an ID like `session-row-001`, `session-row-002` and then change these tests to find by id instead of the crazy/fragile css selectors?